.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
```

Comment:
What's the effect of this if we get a flurry of little PRs?

Reply:
For now we should not hit this limit because you don't have any deps in Go and we just have a hand-ful of Github Actions which need maintenance. But yes, if there's too many deps which need to be updated at the same time, this limits the number of PRs the bot will create.

Reply:
OK